mk/icarus.mk

```diff
@@ -1,11 +1,15 @@
-
 include mk/common.mk
 
 default: lint
 
 .PHONY: lint
-lint: $(addprefix ${OUTDIR}/,$(addsuffix .icarus_lint.txt,${TESTCASE_NAMES}))
+lint: $(addprefix ${OUTDIR}/,$(addsuffix .icarus_lint.stdout,${TESTCASE_NAMES}))
+
+# -tnull: No specific target for compilation; no code generation.
+# -g2012: Select the IEEE1800-2012 standard.
```

Comment:
May be worth mentioning that this is selected because Icarus does not support 2017